CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 2.7.1
+
+### 🐞 Bug fixes
+
+* Work around a [Safari rendering bug](https://bugs.webkit.org/show_bug.cgi?id=237918#c3) by disabling WebGL context antialiasing for Safari 15.4 and 15.5 ([#11615](https://github.com/mapbox/mapbox-gl-js/pull/11615))
```

Comment:
Do users need to know that this is a work around?
```suggestion
* Fix a [Safari rendering bug](https://bugs.webkit.org/show_bug.cgi?id=237918#c3) by disabling WebGL context antialiasing for Safari 15.4 and 15.5 ([#11615](https://github.com/mapbox/mapbox-gl-js/pull/11615))
```

Reply:
This is the current changelog in the tagged release but we can probably edit it. @ansis

Reply:
On the other hand maybe work around is more clear that the bug is Safari's fault, not ours. 🤷